tools/destroy_image.py

```diff
@@ -16,15 +16,11 @@
     if img['Name'] == SNAPSHOT_NAME:
         MEILI_IMG = boto3.resource(
             'ec2', config.AWS_DEFAULT_REGION).Image(id=img['ImageId'])
-        print("Found image: {name} created at {creation_date}".format(
-            name=MEILI_IMG.name,
-            creation_date=MEILI_IMG.creation_date
-        ))
+        print(f'Found image: {MEILI_IMG.name,} created at {MEILI_IMG.creation_date}')
```

Comment:
I think it should be:

```suggestion
        print(f'Found image: {MEILI_IMG.name} created at {MEILI_IMG.creation_date}')
```